eduvpn/metadata.pyi

```diff
@@ -9,15 +9,15 @@ def get_distributed_tokens(): ...
 
 
 class Metadata:
-    api_base_uri = ...  # type: str
+    api_base_uri = ...  #type: str
     profile_id = ...  # type: str
     token = ...  # type: dict
     token_endpoint = ...  # type: str
     authorization_type = ...  # type: str
     authorization_endpoint = ... # type: str
     profile_display_name = ...  # type: str
     two_factor = ...  # type: bool
-    two_factor_method = ... # type: List[str]
+    two_factor_method = ... #type: List[str]
```

Comment:
this is wrong, the space should be there.

Reply:
So this error is repeated in almost every location. pep8/mypy dicates
```
blabla = 'bla bla'  # type: str
```

So 2 spaces before `#` and one after

Reply:
ah also, wasn't this supposed to be moved to the `.py` file and this `.pyi` file removed?

Reply:
Fixed the error and removed the last .pyi files.